packages/web3-constants/evm/savings.json

```diff
@@ -58,5 +58,68 @@
         "Fantom": "",
         "Aurora": "",
         "Aurora_Testnet": ""
+    },
+
+    "AAVE_SUBGRAPHS": {
+        "Mainnet": "https://api.thegraph.com/subgraphs/name/aave/protocol-v2",
+        "Ropsten": "",
+        "Rinkeby": "",
+        "Kovan": "https://api.thegraph.com/subgraphs/name/aave/protocol-v2-kovan",
+        "Gorli": "",
+        "BSC": "",
+        "BSCT": "",
+        "Matic": "",
+        "Mumbai": "",
+        "Arbitrum": "",
+        "Arbitrum_Rinkeby": "",
+        "xDai": "",
+        "Avalanche": "",
+        "Avalanche_Fuji": "",
+        "Celo": "",
+        "Fantom": "",
+        "Aurora": "",
+        "Aurora_Testnet": ""
+    },
+
+    "AAVELendingPoolAddressProviderContract": {
+        "Mainnet": "0xb53c1a33016b2dc2ff3653530bff1848a515c8c5",
+        "Ropsten": "",
+        "Rinkeby": "",
+        "Kovan": "​​0x88757f2f99175387ab4c6a4b3067c77a695b0349",
+        "Gorli": "",
+        "BSC": "",
+        "BSCT": "",
+        "Matic": "",
+        "Mumbai": "",
+        "Arbitrum": "",
+        "Arbitrum_Rinkeby": "",
+        "xDai": "",
+        "Avalanche": "",
+        "Avalanche_Fuji": "",
+        "Celo": "",
+        "Fantom": "",
+        "Aurora": "",
+        "Aurora_Testnet": ""
+    },
+
+    "AAVE": {
```

Comment:
Why not move this to `packages/web3-constants/evm/token.json`?

Reply:
Just following how LIDO implemented theirs

Reply:
If you don't have any other concern bout this, i will consider it resolved